src/app.js

```diff
@@ -1 +1,12 @@
+/* eslint-disable max-len */
```

Comment:
Avoid disabling eslint rules if possible. If the line is too long, consider breaking it into multiple lines or rephrasing it to fit within the max length.